nextjs-app/package.json

```diff
@@ -6,11 +6,12 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint"
+    "lint": "next lint",
+    "wdio:next": "../node_modules/.bin/wdio run ./wdio.conf.js"
```

Comment:
Allows us to call the `wdio` executable in the root node_modules, without having to install WDIO dependencies for each application.